bin/buildrelease.sh

```diff
@@ -56,7 +56,7 @@ echo "Updating version string ..."
 perl -p -i -e "s{__version__ =.*}{__version__ = '$TAG'}g" src/python/WMCore/__init__.py
```

Comment:
Since you're making changes, how about changing this line and get rid of perl. You can replace this with sed and use flag `-i` which will do replacement in place of given file. Getting rid of perl will reduce dependency chain and shell script can rely on standard UNIX tools rather dedicated programming language.